v3/docs/gs/sysreqs.md

```diff
@@ -8,18 +8,25 @@ The following are system requirements for running HSE applications.
 Hardware requirements are largely dictated by the application embedding HSE
 and the amount of data stored.  The following are general guidelines.
 
-* **Architecture**: 64-bit Intel&reg; and AMD (x86_64); 64-bit IBM Z&reg; (s390x)
+* **Architecture**: 64-bit Intel&reg; or AMD (x86_64); 64-bit IBM Z&reg; (s390x)
```

Comment:
you might just mention that HSE only supports 64 bit instead of repeating it 3 times.

Reply:
Done